templates/azure/index.html

```diff
@@ -22,25 +22,20 @@
           <h1>Ubuntu on Azure</h1>
         </div>
         <div class="p-section--shallow">
-          <p>
-            Canonical produces official Ubuntu images in the Microsoft Azure Marketplace and continuously tracks and delivers updates to these, ensuring security and stability are built-in from the moment your machines and containers launch.
-          </p>
-          <p>
-            Canonical works closely with Microsoft to optimise Ubuntu images on Azure and ensure Ubuntu supports the latest cloud features as these are released. Today, Ubuntu powers more mission-critical workloads on Azure than any other operating system.
-          </p>
-          <p>
-            Beyond Ubuntu Server images, Canonical offers a number of premium Ubuntu images that extend Ubuntu Server with additional security and management features, billed through your existing Microsoft Azure account.
-          </p>
+          <p>Canonical produces official Ubuntu images on Microsoft Azure with continuous updates to ensure security and stability from the moment your machines and containers launch.</p>
+          <p>Microsoft works closely with Canonical to optimize Ubuntu images on Microsoft Azure and ensure Ubuntu supports the latest cloud features as these are released. Today, Ubuntu powers a large number of mission-critical workloads on Azure.</p>
+          <p>Beyond Ubuntu, Canonical also offers Charmed Apps on Microsoft Azure, enabling hybrid environments.</p>
         </div>
         <hr class="p-rule--muted" />
-        <a href="/azure/contact-us" class="p-button--positive js-invoke-modal">Talk to us about Ubuntu on Azure</a>
+        <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.ubuntu-24_04-lts?tab=Overview" class="p-button--positive">Launch Ubuntu 24.04 on Azure</a>
+        <a href="/azure/contact-us" class="js-invoke-modal">Talk to us about Ubuntu on Azure&nbsp;&rsaquo;</a>
```

Comment:
You could use the [Vanilla cta block](https://vanillaframework.io/docs/patterns/cta-block) here

Reply:
Done